README.md

```diff
@@ -1,2 +1,53 @@
-# reviewapps-deploy-status
-A Github Action that tests the deploy status of a Heroku Review App
+# Review Apps deployment status
+A Github Action that tests the deploy status of a Heroku Review App.
+
+## Usage:
+* Include the action in the workflow
+    ```yaml
+    name: Review App Test
+
+    on:
+        pull_request:
+            branches:
+            - master
+
+    jobs:
+    review-app-test:
+
+        runs-on: ubuntu-latest
+        
+        steps:
+        - uses: actions/checkout@v1
+        - name: Run review-app test
+        uses: #TODO: After publishing the action
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+            interval: 10 # optional in seconds
+            accepted_responses: 200 # default
```

Comment:
I would make both parameters optional (the current comments say otherwise) then have the text something like this:

```
        with:
            interval: 10 # in seconds, optional, default is 10
            accepted_responses: 200 # list of ints, optional, default is 200
```

Reply:
Thanks for pointing out. Fixed.